hfdata.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+import pyarrow.parquet as pq
+
+def row_to_article_string(row): 
+    '''Convert a row that represents an SEP article to a string'''
+
+    article_string = row['title'] + ''.join(row['preamble'])
```

Comment:
Is the preamble sentences with punctuation at the end? Checking whether we should join with a space or empty string. Similar check elsewhere.

Reply:
Ya so in general each of the content sections (preamble, various sections etc.) is like a list of paragraphs where each para starts with a `\n` and ends with some punctuation and no spaces. So should get something that basically looks like the original article if you handle it in the way i've done here. Can add some checks to make sure